code/src/Artists.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import data from './data.json'
+
+export const Artists = () => {
+    return (
+        <div>
+            <data />
+            Export Artists!
+        </div>
+    )
+}
+
```

Comment:
Looks like this file isn't used (and react doesn't know how to render json, so it'd break if it was!). Don't forget to remove unused code.